mzLib/FlashLFQ/ChromatographicPeak.cs

```diff
@@ -68,6 +73,13 @@ public static string TabSeparatedHeader
             }
         }
 
+        public void SetRtWindow(double rtHypothesis, double? rtStdDev, double? rtInterquartileRange)
```

Comment:
I don't know why this is public or why you need to set all three values at once. Seems like it should be a private method and there should be interdepencies or whatever.